.vuepress/theme/styles/theme.styl

```diff
@@ -54,6 +54,26 @@ body
   border-right 1px solid $borderColor
   overflow-y auto
 
+.banner
```

Comment:
Not quite sure why but you do have a small bug that once the cookie notice is accepted there is white space between the new banner and the hero image which looks weird: 
![image](https://github.com/systemetric/robocon-website/assets/13338287/cbf6f64d-bfc4-4d4a-a4b2-11301d8203a2)

Reply:
ill try to fix that now thx

Reply:
should be fixed now just moved the img up a bit